pom.xml

```diff
@@ -2512,7 +2512,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-lang3</artifactId>
-                <version>3.3.2</version>
+                <version>3.6</version>
```

Comment:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14197 (3.6)

Reply:
this was baselined on hadoop before FYI

Reply:
GeoServer has commons-lang-2.6.  (I.e., it should not overlap)

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16004